bodhi/tests/client/test_bindings.py

```diff
@@ -1182,6 +1182,17 @@ def test_update_with_anon_comment(self):
                 "Comments: bodhi ",
                 "Comments: bodhi (unauthenticated) ")))
 
+    @mock.patch.dict(
+        client_test_data.EXAMPLE_UPDATE_MUNCH, {u'notes': u'This note contains `unicode` char'})
+    def test_update_with_unicode_note(self):
+        """Ensure unicode content in update notes is correctly handled"""
+        client = bindings.BodhiClient()
+        client.base_url = 'http://example.com/tests/'
+
+        text = client.update_str(client_test_data.EXAMPLE_UPDATE_MUNCH)
+
+        self.assertIn(u'Notes: This note contains `unicode` char', text)
```

Comment:
I modified this test a bit to make sure it generates the same traceback as reported in the original ticket:

```
diff --git a/bodhi/tests/client/test_bindings.py b/bodhi/tests/client/test_bindings.py
index 6cd8e2df..900c82a5 100644
--- a/bodhi/tests/client/test_bindings.py
+++ b/bodhi/tests/client/test_bindings.py
@@ -1183,7 +1183,8 @@ class TestBodhiClient_update_str(unittest.TestCase):
                 "Comments: bodhi (unauthenticated) ")))

     @mock.patch.dict(
-        client_test_data.EXAMPLE_UPDATE_MUNCH, {u'notes': u'This note contains `unicode` char'})
+        client_test_data.EXAMPLE_UPDATE_MUNCH,
+        {u'notes': u'This note contains a unicode char ☺'})
     def test_update_with_unicode_note(self):
         """Ensure unicode content in update notes is correctly handled"""
         client = bindings.BodhiClient()
@@ -1191,7 +1192,7 @@ class TestBodhiClient_update_str(unittest.TestCase):

         text = client.update_str(client_test_data.EXAMPLE_UPDATE_MUNCH)

-        self.assertIn(u'Notes: This note contains `unicode` char', text)
+        self.assertIn(u'Notes: This note contains a unicode char ☺', text)

     @mock.patch.dict(
         client_test_data.EXAMPLE_UPDATE_MUNCH, {u'alias': None})
```

Reply:
This test causes this error message if you run it against the unmodified bindings:

```
        if update['notes']:
>           buf = six.moves.cStringIO(update['notes'])
E           UnicodeEncodeError: 'ascii' codec can't encode character u'\u263a' in position 34: ordinal not in range(128)                                                                                                                     

bodhi/client/bindings.py:800: UnicodeEncodeError
```